libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/dh-processes-tab-content.component.ts

```diff
@@ -46,9 +46,8 @@ export class DhProcessesTabContentComponent implements OnDestroy {
     this.loadProcessData();
   }
 
-  processes$: Observable<Process[]> = this.store.processes$;
+  processes$ = this.store.processes$;
   isLoading$ = this.store.isLoading$;
-  processesNotFound$ = this.store.processesNotFound$;
```

Comment:
question: Is showing a "not found" error message part of another PR?

Reply:
That was the idea. It's currently a TODO in the table component.
![image](https://user-images.githubusercontent.com/22943367/157022921-11198c8d-ef54-4599-89a0-01ca42992d56.png)